packages/beelzebub/data_stream/logs/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,391 @@
+      value: event
+
+  - append:
+      field: event.category
+      value: network
+
+  - append:
+      field: event.type
+      value: info
+
+  - rename:
+      if: ctx.event?.original == null
+      description: "Renames the original `message` field to `event.original` to store a copy of the original message. The `event.original` field is not touched if the document already has one; it may happen when Logstash sends the document."
+      field: message
+      target_field: event.original
+      ignore_missing: true
+      tag: rename_message_to_event_original
+
+  - remove:
+      if: ctx.event?.original != null
+      description: "Remove unnecessary message field if event.original exists"
+      field: message
+      ignore_missing: true
+
+  - redact:
+      if: 'ctx.tags != null && ctx.tags.contains("redact_passwords") && !(ctx.event?.original =~ /\"Password\": *\"\"/)'
+      description: "Redact any passwords in event.original"
+      field: event.original
+      patterns:
+        - '"Password": *"%{DATA:REDACTED}"'
+      prefix: '"Password": "<'
+      suffix: '>"'
+      ignore_missing: true
+      tag: redact_event_original
+
+  - remove:
+      description: "Remove filebeat parsed json object if redaction occurred"
+      if: 'ctx.tags != null && ctx.tags.contains("redact_passwords") && !(ctx.event?.original =~ /\"Password\": *\"\"/)'
+      field: json
+      ignore_missing: true
+      tag: remove_json_due_to_redaction
+
+  - json:
+      if: ctx.event?.original != null && ctx.json == null
+      description: "Parse JSON from event.original only if we don't already have a JSON object provided by filebeat, or if redaction occurred"
+      field: event.original
+      target_field: json
+      tag: json_event_original
+
+  - date:
+      if: ctx.json?.event?.DateTime != null
+      description: "ECS mapping json.event.DateTime -> @timestamp, this is likely to be more accurate than json.time"
+      field: json.event.DateTime
+      formats:
+        - ISO8601
+      tag: parse_json_event_datetime
+
+  - date:
+      if: ctx.json?.event?.DateTime == null && ctx.json?.time != null
+      description: "ECS mapping json.time -> @timestamp, only if json.event.DateTime does not exist"
+      field: json.time
+      formats:
+        - ISO8601
+      tag: parse_json_time
+
+  - set:
+      description: "ECS mapping json.event.ID -> event.id"
+      field: event.id
+      copy_from: json.event.ID
+      ignore_empty_value: true
+
+  - set:
+      description: "ECS mapping json.event.Msg -> event.reason, this is likely to be more detailed than json.msg"
+      field: event.reason
+      copy_from: json.event.Msg
+      ignore_empty_value: true
+
+  - set:
+      description: "ECS mapping json.msg -> event.reason, only if json.event.Msg does not exist"
+      field: event.reason
+      copy_from: json.msg
+      ignore_empty_value: true
+
+  - convert:
+      if: ctx.json?.event?.SourceIp != null && ctx.json?.event?.SourceIp != ""
```

```suggestion
      if: ctx.json?.event?.SourceIp != null && ctx.json.event.SourceIp != ""
```
no need to check twice.